cmd/sops/main.go

```diff
@@ -428,7 +454,8 @@ func getKeySources(c *cli.Context, file string) ([]sops.KeySource, error) {
 	}
 	kmsKs := sops.KeySource{Name: "kms", Keys: kmsKeys}
 	pgpKs := sops.KeySource{Name: "pgp", Keys: pgpKeys}
-	return []sops.KeySource{kmsKs, pgpKs}, nil
+	cloudKmsKs := sops.KeySource{Name: "cloud-kms", Keys: cloudKmsKeys}
```

Comment:
I think we should use `cloud_kms` for the key source name, as we currently use `snake_case` for YAML keys (e.g. `unencrypted_suffix`).